[FIX] account_statement_import_txt_xlsx: Raise UserError instead of continue on currency mismatch

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/account-financial-tools#822

rov-adhoc committed:
[FIX] account_statement_import_txt_xlsx: Raise UserError instead of continue on currency mismatch

X-original-commit: ffb1f90f8d6495bbc70cf59b5594b2479183d7d7

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-financial-tools/pull/839.png)](https://mergebot.adhoc.ar/ingadhoc/account-financial-tools/pull/839)

fw-bot-adhoc commented:
@rov-adhoc @maq-adhoc cherrypicking of pull request ingadhoc/account-financial-tools#822 failed.

stdout:
```
CONFLICT (modify/delete): account_statement_import_txt_xlsx/models/account_statement_import.py deleted in 2a7d4aba93e4c13e4d69adf29de4946237a75dc4 and modified in 08c81655a1c130d8bea455455cdb19f76541f556.  Version 08c81655a1c130d8bea455455cdb19f76541f556 of account_statement_import_txt_xlsx/models/account_statement_import.py left in tree.
CONFLICT (modify/delete): account_statement_import_txt_xlsx/models/account_statement_import_sheet_parser.py deleted in 2a7d4aba93e4c13e4d69adf29de4946237a75dc4 and modified in 08c81655a1c130d8bea455455cdb19f76541f556.  Version 08c81655a1c130d8bea455455cdb19f76541f556 of account_statement_import_txt_xlsx/models/account_statement_import_sheet_parser.py left in tree.
CONFLICT (modify/delete): account_statement_import_txt_xlsx/tests/test_account_statement_import_txt_xlsx.py deleted in 2a7d4aba93e4c13e4d69adf29de4946237a75dc4 and modified in 08c81655a1c130d8bea455455cdb19f76541f556.  Version 08c81655a1c130d8bea455455cdb19f76541f556 of account_statement_import_txt_xlsx/tests/test_account_statement_import_txt_xlsx.py left in tree.

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

:warning: after resolving this conflict, you will need to merge it via @roboadhoc.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
